users/views.py

```diff
@@ -81,4 +83,78 @@ def post(self, request):
             return JsonResponse({'message':'KEY_ERROR'}, status=400)
         
         except User.DoesNotExist:
-            return JsonResponse({'message':'INVALID_USER'}, status=401)
+            return JsonResponse({'message':'INVALID_USER'}, status = 401)
+
+
+class CartView(View):    
+    @signin_decorator
+    def post(self, request):
+        
+        try:
+            data         = json.loads(request.body)
+            product_name = data['product_name']
+            sizeup       = data['sizeup']
+
+            if sizeup:            
+                if not Product.objects.filter(relative_product__name = product_name).exists():
+                    return JsonResponse({'message': 'SIZEUP_INVALID'}, status = 406)
+
+            product = Product.objects.get(relative_product__name = product_name)
+
+            cart, created = Cart.objects.get_or_create(user = request.user, product_id = product.id, defaults={'quantity': 1})
```

Comment:
이 부분은 반복해서 말씀드리는 것 같네요~
추가로 궁금한점이 있습니다.
- `quantity`를 1로 잡은 이유가 혹시 수량을 받아서 장바구니에 추가하는게 아니라 오로지 1개씩만 담을 수 있는 구조여서 이렇게 작성하신걸까요?
```suggestion
            cart, created = Cart.objects.get_or_create(user = request.user, product = product, defaults={'quantity': 1})
```